src/cesiumutils.ts

```diff
@@ -284,6 +284,33 @@ export function projectPointOnSegment(point, startPoint, endPoint, start, end, h
   return updateHeightForCartesianPositions([position], height)[0];
 }
 
+/**
+ * Gets distance to point in percentage and returns cartesian position on the line
+ *
+ * @param linePositions
+ * @param ratio - distance to point in percentage (value from 0 to 1 where 0 first point of the line and 1 is last)
+ * @param result
+ */
+export function getPointOnPolylineByRatio(linePositions: Cartesian3[], ratio: number, result = new Cartesian3()) {
```

Comment:
```suggestion
export function getPointOnPolylineByRatio(linePositions: Cartesian3[], ratio: number, result)) {
```
Let's require the result parameter since not passing it does not work anyway.